### ISSUE TITLE
A11y_AzureDataStudio_New Deployment: SQL Server 2019 Big Data Cluster_Screen Reader: Narrator is not announcing the label of Accept terms of use.

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/azuredatastudio#18592]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info & Relationships; #Narrator; #NVDA; #Element:Label;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on the Narrator.
2.Launch Azure Data Studio.
3.Select the . Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment".
5. In the Deployment options, select "SQL Server Big Data Cluster".
6. In Step-1 of Deployment pre-requisites there is a labels of above checkbox.
7.Verify whether that the narrator is announcing the Labels of Accept terms of use or not.
**Note:**
Similar issue can be observed in the Label of Target Database in the Page of Select Deploy Dacpac settings 
### Actual Result:
Narrator is not announcing the label of Accept terms of Use.

### Expected Result:
Narrator should announced the label of Accept terms of use.

### Attachment:
**NVDA Attachment:**
**![A11y_AzureDataStudio_New Deployment_SQL server 2019 Big Data Cluster_NVDA](https://user-images.githubusercontent.com/99962579/156149542-548e84ba-9b5f-43e2-8ee8-a868d7f51cbb.gif)**
**Narrator Attachment:**
**![A11y_AzureDataStudio_New Deployment SQL server 2019 Big Data Cluster_Labels_Narrator](https://user-images.githubusercontent.com/99962579/156148738-b4b8b376-64e7-451b-935f-1af973dcf477.gif)**

**Comment by Charles-Gagnon** [comment]
This is text so is not read by default. (it is a header for the section) 

f you enter text scan mode then you can browse the text on the page and it will read it off correctly.

**Comment by Gayathri1501** [comment]
Verified this issue in scan mode. Issue is Fixed. Hence, closing this bug.
![A11y_AzureDataStudio_New Deployment_ SQL Server 2019 Big Data Cluster_Screen Reader_](https://user-images.githubusercontent.com/99962579/178236569-2bd413b8-962f-45c1-a9ad-a5d8a1f836d9.gif)

**Comment by SachinChavhan317** [comment]
Adding close tag. #Closed

**Comment by Samhitha66** [comment]
#Closed;

**Comment by mstechie** [comment]
#BM-AzureDataStudio-Win32-Feb2022;
